source/articles/releasenotes/Unreleased-Changes.md

```diff
@@ -6,3 +6,60 @@ This page contains changes and fixes that were not inluded in any release yet an
 
 none yet
 
+## ASP.NET Core support
+LINQ To DB now has support for ASP.NET Dependancy injection. Here's a simple example of how to add it to dependancy injection
+```C#
+public class Startup
+{
+    public IConfiguration Configuration { get; }
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        //...
+        //using LinqToDB.AspNet
+        services.AddLinqToDbContext<AppDataConnection>((provider, options) => {
+            options
+            //will configure the AppDataConnection to use
+            //SqlServer with the provided connection string
+            //there are methods for each supported database
+            .UseSqlServer(Configuration.GetConnectionString("Default"))
+
+            //default logging will log everything using
+            //an ILoggerFactory configured in the provider
+            .UseDefaultLogging(provider);
+        });
+        //...
+    }
+}
+```
+
+In addition to these configuration options the following are also supported
+* `UseOracle(string connectionString)`
+* `UsePostgreSQL(string connectionString)`
+* `UseMySql(string connectionString)`
+* `UseSQLite(string connectionString)`
+* `UseConnectionString(string providerName, string connectionString)`
+* `UseConnectionString(IDataProvider dataProvider, string connectionString)`
+* `UseConfigurationString(string configurationString)`
+* `UseConnectionFactory(IDataProvider dataProvider, Func<IDbConnection> connectionFactory)`
+* `UseConnection(IDataProvider dataProvider, IDbConnection connection, bool disposeConnection = false)`
+* `UseTransaction(IDataProvider dataProvider, IDbTransaction transaction)`
+
+We've done our best job to allow any existing use case to be migrated to using the new configuration options, please create an issue if something isn't supported.
```

Comment:
maybe it makes sense to add that we will add more Use<provider_name> methods in future, so people not get confused by only 5 of 12 providers available.